specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -436,13 +504,17 @@ scope kafka
         struct KafkaOffsetCommitBeginEx
         {
             string16 groupId;
-            string16 topic;
+            int32 generationId;
+            string16 memberId;
         }
 
         struct KafkaOffsetCommitDataEx
         {
+            string16 topic;
```

Comment:
The idea is to have one offset commit stream per group instead of one per topic?